resources/src/IconActionHelp.tsx

```diff
@@ -0,0 +1,94 @@
+import { useContext } from 'react'
+import { ArrowDownCircle, Filter, HelpCircle, Search } from '@geist-ui/react-icons'
+import { Link, Modal, Text, Tooltip, useModal } from '@geist-ui/react'
+import { DataContext } from './Data'
+import { useEscClose } from './useEscClose'
+
+const UpdateTime = () => {
+  const { data } = useContext(DataContext)
+  return <span>{data?.last_update}</span>
+}
+
+const HelpContent = () => (
+  <>
+    <Text h3>the tldr-pages project</Text>
+    <Text p>
+      The{' '}
+      <Link icon href="https://tldr.sh/" target="_blank">
+        tldr-pages project
+      </Link>{' '}
+      provides a collection of community-maintained help pages for command-line tools.
+    </Text>
+    <Text h3>this website</Text>
+    <Text p>
+      The main feature of this website is a table which lists the translation status for all tldr
+      pages. You can click every status icon (legend) and you'll be taken to the respective page on
+      GitHub or to a dialogue to create a new one.
+    </Text>
+    <Text p>
+      The pages are grouped by operating system and are sorted in alphabetical order. The blue rows
+      mark a new group for an operating system and show the percentage of pages translated in this
+      group (an outdated page still counts as translated).
+    </Text>
+    <Text p>
+      The underlying dataset is based on the{' '}
+      <Link icon href="https://github.com/tldr-pages/tldr/" target="_blank">
+        main
+      </Link>
+      branch of tldr-pages/tldr repository and is scheduled to be updated every day at 0am UTC. The
```

Comment:
```suggestion
      branch of tldr-pages/tldr repository and is scheduled to be updated every day at 12am UTC. The
```
or
```suggestion
      branch of tldr-pages/tldr repository and is scheduled to be updated every day at 0 UTC. The
```
or
```suggestion
      branch of tldr-pages/tldr repository and is scheduled to be updated every day at midnight UTC. The
```
?

Reply:
Good point, I would prefer the wording `midnight`